### ISSUE TITLE
Helidon init does not produce app.yaml for 3.x.x

### ISSUE THREAD

**Issue by DavideRossi** [helidon-io/helidon-build-tools#997]
It used to work (and it's also described in the quickstart).
For some reason now it only works when selecting version 4.0.0.
Creating a project with the Maven archetype does not work either (which I guess is what helidon init does under the hood) but I admit I did not try that in the past.

**Comment by tvallin** [comment]
Indeed, this issue was reported and fixed under the Helidon repository: https://github.com/helidon-io/helidon/issues/7215

**Comment by DavideRossi** [comment]
I see. Still does not work with the current version of helidon init when selecting a 3.x.x version. Is there a way to fix this behavior?

**Comment by tvallin** [comment]
Yes, it will be included in the next 3.x Helidon release.

**Comment by DavideRossi** [comment]
Yet there is something weird to my eyes.
Creating a 3.x.x project results in a missing app.yaml while creating a 4.0.0 project with very same version of the helidon cli does not. This seems to suggest that the helidon cli is using different revisions of `archetypes/helidon/src/main/archetype/common/presets.xml` depending on the chosen Helidon version.
And, if that is the case, why it "sees" a different preset.xml for 3.x.x than what it used to "see" a couple of versions ago (given that it used to create the app.yaml file)?
